src/pages/LoginPage.jsx

```diff
@@ -1,7 +1,148 @@
-import React from "react";
+import React, { useState } from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+
+const LoginSchema = Yup.object().shape({
+  email: Yup.string()
+    .email('Invalid email address')
+    .required('Email is required'),
+  
+  password: Yup.string()
+    .min(6, 'Password is too short!')
+    .required('Password is required'),
+});
+
+const RegisterSchema = Yup.object().shape({
+  username: Yup.string()
+    .min(3, 'Username is too short!')
+    .max(50, 'Username is too long!')
+    .required('Username is required'),
+  
+  email: Yup.string()
+    .email('Invalid email address')
+    .required('Email is required'),
+  
+  password: Yup.string()
+    .min(6, 'Password is too short!')
+    .required('Password is required'),
+  
+  confirmPassword: Yup.string()
+    .oneOf([Yup.ref('password'), null], 'Passwords must match')
+    .required('Confirm password is required'),
+});
+
 
 const LoginPage = () => {
-  return <div>LoginPage</div>;
+  const [isLogin, setIsLogin] = useState(true);
+
+  const handleSubmit = (values, { setSubmitting }) => {
+    setTimeout(() => {
```

Comment:
why you use settimout please remove it and make a rest for the form when it is submitted